packages/common/package.json

```diff
@@ -43,7 +43,7 @@
     "prettier": "prettier --write 'src/**/*.ts'"
   },
   "dependencies": {
-    "@pdfme/pdf-lib": "^1.17.3",
+    "@pdfme/pdf-lib": "^1.18.1",
```

Comment:
Since this version, we can use `page.drawSvg`
ref: https://github.com/pdfme/pdf-lib/pull/3